cl/phase1/network/services/constants.go

```diff
@@ -6,13 +6,15 @@ import (
 )
 
 const (
-	SeenBlockCacheSize          = 1000 // SeenBlockCacheSize is the size of the cache for seen blocks.
-	blockJobsIntervalTick       = 50 * time.Millisecond
-	blobJobsIntervalTick        = 5 * time.Millisecond
-	attestationJobsIntervalTick = 100 * time.Millisecond
-	blockJobExpiry              = 7 * time.Minute
-	blobJobExpiry               = 7 * time.Minute
-	attestationJobExpiry        = 30 * time.Minute
+	validatorAttestationCacheSize = 100_000
```

Comment:
for now # of active validators is almost 1million. should we adjust this value to, like 500k?
https://beaconcha.in/charts/validators

Reply:
no, while true, the actual useful attestations is a low number. something like 50k-75k.